src/python/pants/engine/isolated_process.py

```diff
@@ -86,11 +87,61 @@ def _verify_env_is_dict(cls, env):
 
 
 class ExecuteProcessResult(datatype(['stdout', 'stderr', 'exit_code', 'output_directory_digest'])):
```

Comment:
`ExecuteProcessResult` probably doesn't have an `exit_code` any more, as it's guaranteed to always be 0